examples/basic/main.tf

```diff
@@ -18,7 +19,7 @@ module "slz_vpc" {
   source            = "../../"
   resource_group_id = module.resource_group.resource_group_id
   region            = var.region
-  name              = var.name
+  name              = "${var.prefix}-slz-vpc"
```

Comment:
no this does't make sense. when you pass a value for prefix, it uses the prefix and name. So var.prefix should not be used here. Can you just set it to `name = vpc` ?